lib/wahwah/mp3_tag.rb

```diff
@@ -72,7 +73,7 @@ def parse_duration
         @bitrate = (bytes_count * 8 / @duration / 1000).round unless @duration.zero?
       else
         @bitrate = mpeg_frame_header.frame_bitrate
-        @duration = (file_size - (@id3_tag&.size || 0)) * 8 / (@bitrate * 1000).to_f unless @bitrate.zero?
+        @duration = (file_size - (@id3_tag&.size || 0)) * 8 / (@bitrate * 1000).to_f unless @bitrate.blank? || @bitrate.zero?
```

Comment:
`@bitrate` under certain circumstances can come back as `nil` -- the case I found was when `layer` ended up being `nil` (I looked at the bits and it looked like it was `00`, or the reserved layer), which means `kind` ends up being `nil`, which means `FRAME_BITRATE_INDEX[kind]` is `nil`, and so `frame_bitrate = FRAME_BITRATE_INDEX[kind]&.fetch(@header[16..19].to_i(2))` is also `nil`.